satpy/etc/readers/aws1_mwr_l1c_nc.yaml

```diff
@@ -0,0 +1,375 @@
+reader:
+  name: aws1_mwr_l1c_nc
+  short_name: AWS L1C RAD NetCDF4
+  long_name: AWS L1C Radiance (NetCDF4)
```

Comment:
Missing MWR.
```suggestion
  short_name: AWS/MWR L1C RAD NetCDF4
  long_name: AWS/MWR L1C Radiance (NetCDF4)
```

Reply:
Do we need "netcdf4" in the short name? It means the "short" one is almost the same length as the "long" one!

Reply:
Thanks, fixed now, will push soon.